packages/ra-core/src/util/index.ts

```diff
@@ -10,7 +10,8 @@ import Ready from './Ready';
 import resolveRedirectTo from './resolveRedirectTo';
 import warning from './warning';
 import useWhyDidYouUpdate from './useWhyDidYouUpdate';
-import { useSafeSetState, useTimeout, useDocumentTitle } from './hooks';
+import useDocumentTitle from './useDocumentTitle';
```

Comment:
We're deprecating `export default` in source files little by little. For new files, you should just export the hooks, components and types, and reexport them as is with:

```tsx
export * from './useDocumentTitle';
```